src/main/java/hometaskStream/WordFinder.java

```diff
@@ -0,0 +1,21 @@
+package hometaskStream;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class WordFinder {
+
+    public static List<String> findWordsByLetter(String text, String letter) {
+        List<String> wordsArray = new ArrayList<>(Arrays.asList(text.toLowerCase().split(" ")));
```

Comment:
please rename this variable to "wordsList" since it is not an array

Reply:
Thanks, done

Reply:
have you pushed your changes?
<img width="1397" alt="image" src="https://user-images.githubusercontent.com/90613721/184429837-40944964-56c4-440e-984d-35be402b4814.png">